src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj

```diff
@@ -4,13 +4,14 @@
   <PropertyGroup>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
     <SkipValidatePackage>true</SkipValidatePackage>
+    <PreReleaseLabel>servicing</PreReleaseLabel>
```

Comment:
This package shouldn't need to be marked with "servicing" as all the packages that build in the release build are already marked as "preview1" as they should (https://github.com/dotnet/corefx/blob/release/1.1.0/Packaging.props#L3).

The ones I thought we wanted to mark as servicing were the ones we are building in TFS. I'm not entirely sure we actually care to do that now that I"m looking at it. We should just keep them as "beta" as they are and then move them to stable once we are done with them.